[2022-06-20 21:39:52,542] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl-processing.transform manual__2022-06-20T21:39:41.843867+00:00 [queued]>
[2022-06-20 21:39:52,591] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: etl-processing.transform manual__2022-06-20T21:39:41.843867+00:00 [queued]>
[2022-06-20 21:39:52,599] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-20 21:39:52,606] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-20 21:39:52,607] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-20 21:39:52,750] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): transform> on 2022-06-20 21:39:41.843867+00:00
[2022-06-20 21:39:52,759] {standard_task_runner.py:52} INFO - Started process 5972 to run task
[2022-06-20 21:39:52,782] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'etl-processing', 'transform', 'manual__2022-06-20T21:39:41.843867+00:00', '--job-id', '728', '--raw', '--subdir', 'DAGS_FOLDER/etl_processing.py', '--cfg-path', '/tmp/tmpjgtvtt7v', '--error-file', '/tmp/tmpvgaq2c90']
[2022-06-20 21:39:52,785] {standard_task_runner.py:80} INFO - Job 728: Subtask transform
[2022-06-20 21:39:53,124] {task_command.py:370} INFO - Running <TaskInstance: etl-processing.transform manual__2022-06-20T21:39:41.843867+00:00 [running]> on host fe209149cfc1
[2022-06-20 21:39:53,998] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=etl-processing
AIRFLOW_CTX_TASK_ID=transform
AIRFLOW_CTX_EXECUTION_DATE=2022-06-20T21:39:41.843867+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-20T21:39:41.843867+00:00
[2022-06-20 21:39:54,270] {python.py:173} INFO - Done. Returned value was: None
[2022-06-20 21:39:54,425] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=etl-processing, task_id=transform, execution_date=20220620T213941, start_date=20220620T213952, end_date=20220620T213954
[2022-06-20 21:39:54,656] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-20 21:39:55,317] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
